Applications/Games/League of Legends/Online/script.js

```diff
@@ -94,12 +79,17 @@ var installerImplementation = {
             })
             .category("Games")
             .wineDistribution("staging")
-            .wineVersion(LATEST_STAGING_VERSION)
+            .wineVersion(LATEST_STAGING_VERSION) // TODO: Gallium9 patchset
             .preInstall(function (wine /*, wizard*/) {
                 wine.windowsVersion("winxp");
                 wine.d3dx9();
-                wine.overrideDLL().set("native, builtin", ["atl120", "msvcp120", "msvcr120", "vcomp120", "msvcp140"]).do();
+                // wine.overrideDLL().set("native, builtin", ["atl120", "msvcp120", "msvcr120", "vcomp120", "msvcp140"]).do(); // Obsolete?
                 wine.enableCSMT();
+                wine.disableGLSL(); // Needs approval + recommended by WineHQ
+                wine.corefonts(); // Needs approval + mandatory for fonts
+                wine.vcrun2008(); // Needs approval
+                wine.vcrun2017(); // Needs approval
```

Comment:
Are they both needed ?

Reply:
Based on my tests they are i've noticed lower latency in-game with both versions.

Reply:
You're missing the imports for both `vcrun2008` and `vcrun2017` at the beginning of the script

Reply:
`missing the imports` more info?

Reply:
At the beginning of the script file you have a bunch of rows looking like the following:
```javascript
include(["engines", "wine", "verbs", "d3dx9"]);
```

These lines import functionality for your script (mainly extension methods for `wine`).
Before you can use most verbs (e.g. `vcrun2008` and `vcrun2017`) you need to import them.
For this you can take a look at the [Verbs](https://github.com/PhoenicisOrg/scripts/tree/master/Engines/Wine/Verbs) folder and other scripts.

Reply:
Just as a note: our documentation doesn't seem to contain an explanation about importing functionality as of now, see #785.

Reply:
Checking will add if missing.